### ISSUE TITLE
$interval outputs in the console

### ISSUE THREAD

**Issue by notflip** [alpine-collective/toolkit#9]
When using the $interval methods as follows `<div x-data="searchResults()" x-init="$interval(() => pollServer(), {timer: 1000, forceInterval: true})">`

I see output in the console, can this be disabled? 

```
true
app.5251e11c.js:32 false
app.5251e11c.js:32 clear 32
```

**Comment by KevinBatdorf** [comment]
is there a console.log statement elsewhere in your code? In pollServer maybe?

**Comment by notflip** [comment]
@KevinBatdorf I found the code, it's in alpine-collective/toolkit-interval/cdn.js line 33

```
Alpine.effect(() => {
          console.log(this.autoIntervalTest);
          if (this.autoIntervalTest == void 0 || this.autoIntervalTest) {
            forceInterval ? loop() : requestAnimationFrame(loop);
          } else {
            console.log("clear", autoIntervalLoop);
            clearTimeout(autoIntervalLoop);
          }
        });
```

**Comment by KevinBatdorf** [comment]
I see it there. That's so strange! I don't see those lines in any of the file changes. It looks like maybe something I using to debug locally at some point. But why is it in the build...

Can you use npm instead? Or even just copy the file contents and remove those lines logging. 

I'm not sure how to fix it but I'll try to research a bit more over the weekend. @SimoTod Any ideas?

**Comment by SimoTod** [comment]
Not sure. The library was tagged ages ago. I assume we can just tag 1.0.1 from master and it will disappear.

**Comment by KevinBatdorf** [comment]
Yeah it's been so long. (Hope you're doing well, btw!) Well, any risk in making a new tag? Will it auto publish? I don't remember if this repo had any extra steps. Do you want to tag it?

**Comment by SimoTod** [comment]
I can't remember either. Hahaha. I can have a look tomorrow or Monday (I won't be close to my laptop any time today). We've probably never added any automation, I suspect. 
You definitely need to update the package.json file(s) for npm, tagging on github is just for consistency.

**Comment by notflip** [comment]
Thanks for the good support! Much appreciated.

**Comment by SimoTod** [comment]
@notflip this should be fixed now see 1.0.1
